graphql/schemabuilder/reflect.go

```diff
@@ -794,6 +794,22 @@ func NewSchema() *Schema {
 	}
 }
 
+// Enum registers an enumType in the schema. The val should be a value
+// of the enumType to be registered in the schema, and the enumMap should be
```

Comment:
"should be the default value of the enumType"? or "arbitrary" value? Maybe make it clear that the value is not used, it's just used for type introspection.